constructor/osx/run_installation.sh

```diff
@@ -110,9 +110,10 @@ if ! "$PREFIX/bin/python" -V; then
     exit 1
 fi
 
-# This is unneeded for the default install to ~, but if the user changes the
+# This is not needed for the default install to ~, but if the user changes the
 # install location, the permissions will default to root unless this is done.
 chown -R "$USER" "$PREFIX"
+chown -R "$USER" "${HOME}/.conda"
```

Comment:
I think it might also leave a `.condarc` somewhere if the config file asked for that option. Should we add it here too? e.g.

```suggestion
test -f "${HOME}/.condarc" && chown "$USER" "${HOME}/.condarc"
```

Reply:
When I tested it, `.condarc` had the correct ownership, but better safe than sorry.